src/common/components/entry-index-menu-dropdown/index.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import DropDown, { MenuItem } from "../dropdown";
+import { Global } from "../../store/global/types";
+import { History } from "history";
+import { _t } from "../../i18n";
+import { ToggleType } from "../../store/ui/types";
+import {
+  menuDownSvg,
+} from "../../img/svg";
+
+interface Props {
+  global: Global;
+  history: History;
+  onChangeGlobal: (value: boolean) => void;
+  isGlobal: boolean;
+  toggleUIProp: (what: ToggleType) => void;
+  isActive: boolean;
+}
+
+export const EntryIndexMenuDropdown = (props: Props) => {
+  const { global: { filter, tag }, history, onChangeGlobal, isGlobal } = props;
+
+  let dropDownItems: MenuItem[] = [
+    {
+      label: <span>{_t('entry-filter.filter-global')}</span>,
+      active: tag === "",
+      onClick: () => onTagValueClick('')
+    },
+    {
+      label: <span>{_t('entry-filter.filter-community')}</span>,
+      active: tag === "my",
+      onClick: () => onTagValueClick('my')
+
+    }
+  ]
+
+  if (filter === 'created') {
+    dropDownItems = [
+      ...dropDownItems,
+      // for adding new menu items - example shown below
+      // {
+      //   label: <span>Now</span>,
+      //   active: tag === "right_now",
+      //   onClick: () => console.log('right_now clicked'),
+      // },
+    ]
```

Comment:
@adilrime17 Can we delete this commented code if it is of no use?

Reply:
I have purposely kept this comment so in the future when we add values then we have an example in there and in this way we need not go through the code to understand the complete logic. But if you still want me to remove this then let me know I will remove and update PR as well.